src/KingSystem/Event/CMakeLists.txt

```diff
@@ -3,4 +3,6 @@ target_sources(uking PRIVATE
   evtEvent.h
   evtManager.cpp
   evtManager.h
+  evtInfoData.cpp
```

Comment:
Could you put these two lines between Event and Manager to keep this list alphabetically sorted?